sql/data_model.sql

```diff
@@ -0,0 +1,59 @@
+BEGIN;
+
+-- add pgcrypto extension so that we can use the gen_random_uuid function.
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+DROP TABLE IF EXISTS formdata CASCADE;
+CREATE TABLE formdata(
+  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
+  domain text NOT NULL,
+  received_on timestamp without time zone,
```

Comment:
actually this is the server time

On 12 June 2015 at 22:48, Ben <notifications@github.com> wrote:

> In sql/data_model.sql
> <https://github.com/dimagi/mobile-endpoint/pull/1#discussion_r32353666>:
>
> > @@ -0,0 +1,59 @@
> > +BEGIN;
> > +
> > +-- add pgcrypto extension so that we can use the gen_random_uuid function.
> > +CREATE EXTENSION IF NOT EXISTS pgcrypto;
> > +
> > +DROP TABLE IF EXISTS formdata CASCADE;
> > +CREATE TABLE formdata(
> > +  id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
> > +  domain text NOT NULL,
> > +  received_on timestamp without time zone,
>
> received on is when the user submits the form on the phone correct? do we
> need another column that autogenerates a timestamp when this record is
> created to serve as our server_time
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/mobile-endpoint/pull/1/files#r32353666>.
>



-- 
Simon Kelly
Senior Engineer | Dimagi South Africa